dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/DefaultIcon.java

```diff
@@ -554,18 +746,24 @@ public enum Icons {
 
     @Getter private final String[] keywords;
 
-    Icons(String key, String description, String[] keywords) {
+    @Getter private final boolean custom;
+
+    Icons(String key, String description, String[] keywords, boolean custom) {
```

Comment:
`custom` shouldn't be part of the constructor and `isCustom` should always return false

Reply:
that is applicable now but if we need to save default icon's information in database then we need this flag.